internal/action/edit.go

```diff
@@ -76,7 +76,7 @@ func (s *Action) editGetContent(ctx context.Context, name string, create bool) (
 			return nil
 		}
 		if err := s.find(ctxutil.WithFuzzySearch(ctx, false), nil, name, cb); err == nil {
-			cont, err := termio.AskForBool(ctx, fmt.Sprintf("No secret at '%s'. Found possible match in '%s'. Edit existing entry?", name, newName), true)
+			cont, err := termio.AskForBool(ctx, fmt.Sprintf("Secret does not exist %q. Found possible match in %q. Edit existing entry?", name, newName), true)
```

Comment:
This prints:
> Secret does not exist "tests". Found possible match in "test". Edit existing entry?

Wouldn't it be best like this:
`"Secret %q does not exist. Found possible match with %q. Edit this entry instead?"`

Reply:
yes, much better.